native-assets/NFT/project/SPECIFICATION.md

```diff
@@ -0,0 +1,82 @@
+Table of Contents
+- [Overview](#overview)
+- [Use Cases](#use-cases)
+  - [NFT Contract](#nft-contract)
```

Comment:
Do we need this heading? Will there be other contracts listed here in the future?